api/src/main/java/org/apache/cloudstack/api/command/admin/guest/UpdateGuestOsCmd.java

```diff
@@ -50,7 +50,7 @@ public class UpdateGuestOsCmd extends BaseAsyncCmd {
     @Parameter(name = ApiConstants.ID, type = CommandType.UUID, entityType = GuestOSResponse.class, required = true, description = "UUID of the Guest OS")
     private Long id;
 
-    @Parameter(name = ApiConstants.OS_DISPLAY_NAME, type = CommandType.STRING, required = true, description = "Unique display name for Guest OS")
```

Comment:
JFYI, this will require a change in go sdk [here](https://github.com/apache/cloudstack-go/blob/main/generate/requiredParams.go) to ensure the SDK is backward compatible.

Reply:
noted

Reply:
@vishesh92 created https://github.com/apache/cloudstack-go/pull/109